src/devices/Qmc5883l/Interrupt.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.Qmc5883l
+{
+    /// <summary>
+    /// Interrupt enabling is controlled by register INT_ENB in control register 2. Once the interrupt is enabled, it will flag
```

Comment:
information about register is only useful for developers of the binding not for consumer of this class. I'd recommend removing register specific information